.github/workflows/codspeed.yml

```diff
@@ -16,6 +16,10 @@ jobs:
         with:
           python-version: '3.12'
       - run: python -m pip install .[tests] pytest-codspeed 'numpy<2'
```

Comment:
We should merge this pr, but without this hack.

Perhaps we should use these two things as "sample data" and integrate them into dkist.data.sample like SunPy?

Reply:
I don't really remember how it works in SunPy, but sure. So `dkist.data.sample` would have a function to download these files and we'd call that for running the benchmarks and for the docs build?